### ISSUE TITLE
Enhance wins JavaScript to assign alt text to contributor photo

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#4112]
### Overview

To comply with Web Content Accessibility Guidelines (WCAG), we need to ensure that image alt text on the Wins page is meaningful and appropriate for each image. This issue addresses the need for alt text on the GitHub profile photo in the Wins cards, which are generated using JavaScript.

### Action Items

- [x] Understand the role of these components of the wins page:
  - [x] The main Wins web page: [wins.html](https://github.com/hackforla/website/blob/gh-pages/pages/wins/wins.html), which uses Liquid to include the template for each win card: [wins-card-template.html](https://github.com/hackforla/website/blob/gh-pages/_includes/wins-page/wins-card-template.html)
  - [x] The JavaScript code [wins js](https://github.com/hackforla/website/blob/gh-pages/assets/js/wins.js) running in the Wins web page.pro
- [x]  In [wins-card-template.html](https://github.com/hackforla/website/blob/gh-pages/_includes/wins-page/wins-card-template.html), find the img element with class 'wins-card-profile-img', and add an empty alt attribute (alt='').
- [x] In [wins.js](https://github.com/hackforla/website/blob/gh-pages/assets/js/wins.js), search for the text '.wins-card-profile-img' in order to locate the line of code which sets the "src" attribute for the GitHub profile photo. Add code to set the "alt" attribute of the image equal to "photo of name" where name is the name of the contributor. For example, the alt text for Mary Smith's photo should be "photograph of Mary Smith".
- [x] View the wins page to test the code at mobile, tablet and full-screen widths. Also also follow the "See More" page to check the alt tag on the modal overlay.

### Resources/Instructions

[Wins webpage](https://hackforla.org/wins/)
[wins.html](https://github.com/hackforla/website/blob/gh-pages/pages/wins/wins.html)
[wins js](https://github.com/hackforla/website/blob/gh-pages/assets/js/wins.js)
[wins-card-template.html](https://github.com/hackforla/website/blob/gh-pages/_includes/wins-page/wins-card-template.html)
This issue was created as a follow-up to https://github.com/hackforla/website/issues/2569. See that issue for accessibility resources.

**Comment by ExperimentsInHonesty** [comment]
@roslynwythe Is there are reason why it has the dependency https://github.com/hackforla/website/issues/4088? It seems like they could be worked on at the same time? Am I missing something?

**Comment by jdingeman** [comment]
Removing dependency and marking as ready for priroritization

**Comment by Jaretzbalba** [comment]
Availability: Weekdays 10-12 am, 3 hours
ETA: 3/21/23
